pilot/utils/utils.py

```diff
@@ -116,7 +116,7 @@ def get_os_info():
     }
 
     if os_info["OS"] == "Linux":
-        os_info["Distribution"] = ' '.join(distro.linux_distribution(full_distribution_name=True))
+        os_info["Distribution"] = ' '.join(distro.name(pretty=True))
```

Comment:
`distro.name` already returns a string so we shouldn't try to join it. Joining in fact splits the string to characters, then adds a space between each, like this:

```
$ python
Python 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import distro
>>> ' '.join(distro.name(pretty=True))
'D e b i a n   G N U / L i n u x   1 2   ( b o o k w o r m )'
>>> 
```

Instead we should just use the return value, as is:

```
>>> print(distro.name(pretty=True))
Debian GNU/Linux 12 (bookworm)
```